### ISSUE TITLE
Can't find app within Integrations

### ISSUE THREAD

**Issue by magnetic157** [PeteRager/lennoxs30#58]
Hello,

I installed the repository through HACS and did a full reset of the server.

When go to Configuration -> Integrations I don't have a "Big orange + button" like the documentation mentions, but I do have a blue button for "+ Add Integration".  When I click on that and search for Lennox nothing comes up.

Repository is showing as installed within HACS though.

**Comment by magnetic157** [comment]
HACS Version = 1.15.2
Home Assistant Version = Core2021.10.4

Thank you for the help!

**Comment by PeteRager** [comment]
That is great news.   Do we need to edit / clarify the steps?

**Comment by cmd204** [comment]
Yeah, I'm having the same issue here. I don't have an orange button either and "Lennox" didn't show up in the setup new integration list. Same color and wording as magnetic157. I also got it working by adding directly to the configuration.yaml file.

**Comment by poldim** [comment]
Check to make sure you’re looking in the right integrations area. There are
two: one for HACS and another one for HA.

**Comment by PeteRager** [comment]
I want to get this issue closed out and the docs clarified in the upcoming release this week.I don’t use HACS or have it setup. Perhaps that’s what I need to do.  Anyways, here’s what we have

I think this first section. Is Ok?

“Install HACS then add the custom repository for this project

Go to any of the sections (integrations, frontend, automation).
Click on the 3 dots in the top right corner.
Select "Custom repositories"
Add the URL to the repository: https://github.com/PeteRager/lennoxs30
Select the integration category.
Click the "ADD" button.
Add integration in HACS”

as is this?

“In the HACS panel, go to integrations and click the big orange '+' button.
Search for Lennox iComfort and click Install this repository in HACS.
Restart HA to load the integration into HA.”

Is this step we are concerned with?  There is no UI based config, so in non HACS HA, going to that doesn’t work for me.

“Go to Configuration -> Integrations and click the big orange '+' button. Look for Lennox and click to add it.”

These seems right?

“Add the configuration to configuration.yaml.
Restart HA.”

**Comment by poldim** [comment]
I think the issue is stemming from people not reading the directions closely enough and going to the wrong integrations area.  But it also doesn't help that HACS and core HA config both have sections called integrations.

Maybe it'd be easier to link to a video of someone installing some HACS integration?

![image](https://user-images.githubusercontent.com/16869713/142882699-90f1c34d-1468-45ee-911c-681ed59b1389.png)

**Comment by DanceMore** [comment]
are you sure this is resolved? after adding the repository to HACS (ala the first half of the screenshot), I would expect to go to the second half and be able to find, install, and subsequently configure the Integration.

your second half:

> ![image](https://user-images.githubusercontent.com/79212033/144672564-b63e60fb-8428-4cf7-b69e-8bc5f7ab54e5.png)

I'm expecting to use the `+ Add Integration` button on that screen, something like this:

![image](https://user-images.githubusercontent.com/79212033/144672624-c07a1fc2-0b88-4090-9c42-ee0dc9a54e4f.png)

dropping config values into `configuration.yaml` works, so it's installed anyways somehow.

but I'm pretty sure HomeAssistant _wants_ everyone to stop using raw YAML and support UI based configuration. I cannot figure out how one installs or modifies it from the webUI.

**Comment by PeteRager** [comment]
Thanks for the note,  I believe you are correct.  Config is via YAML, so it will not appear in that second step.  Next release we will have UI based Config 

I haven’t gotten HACS installed yet, so relying on community for this piece.

**Comment by poldim** [comment]
> I'm expecting to use the `+ Add Integration` button on that screen, something like this:
> 
> ![image](https://user-images.githubusercontent.com/79212033/144672624-c07a1fc2-0b88-4090-9c42-ee0dc9a54e4f.png)


Can you post a full (browser) screen of this image?

**Comment by dcj** [comment]
I went through this today. See #119.
IMHO, the instructions say to add this integration through HACS (which is correct), and then also add the integration via HA (which is incorrect), because, as mentioned above, the normal HA/Integrations cannot find/see this.
So, the "fix" would be to remove this in the documentation:
```Add integration in Home Assistant

Go to Home Assistant Configuration Panel -> Integrations (https://your.domain/config/integrations) - click the '+ Add integration' button. Search for 'Lennox' and click to add it.
Add the configuration to configuration.yaml.
Restart HA.
```

**Comment by PeteRager** [comment]
Was automatically closed as changes are made and merged into the upcoming release available by end of year.

The new release supports UI based configuration which solves all these issues.

**Comment by PeteRager** [comment]
https://github.com/PeteRager/lennoxs30/releases/tag/v0.2.0

**Comment by DanceMore** [comment]
thanks and big-ups on the large release !!

I saw the changelog in HACS, it upgraded flawlessly and ingested my Config. I deleted the old now-broken entity and renamed the new one to remove the `_2` or whatever it had appended due to the name conflict of default objects. (I expected this from many moons of running home-assistant, but it may have been listed in your changelog / docs too)

I came to check on my Issue and it was already closed as part of the `v0.2.0` tag.

great work ❤️🧡💛💚💙💜

**Comment by PeteRager** [comment]
Thanks for the positive feedback!  We always hear when things don't work, nice to hear it's working.

Let me know if there are other items we should add in? Next item on my list is to support the Lennox HVAC Alerts the S30s generate (e.g. if something goes wrong with the HVAC system it'd be nice to have these reflected into Home Assistant)
